### ISSUE TITLE
module 'vim.uri' not found

### ISSUE THREAD

**Issue by bc-geoffl** [nvim-lua/kickstart.nvim#454]
## Describe the bug

I try to do a clean install and I fail on this error. 

- I tried 0.10, but am running into this issue.
- it looks like the recommended version is 0.9.1?

## To Reproduce

1. ...
```
% rm -rf ~/.config/nvim
% git clone git@github.com:bc-geoffl/kickstart.nvim.git ~/.config/nvim && nvim ~/.config/nvim/init.lua
Error detected while processing /Users/geoffl/.config/nvim/init.lua:
E5113: Error while calling lua chunk: vim/_init_packages.lua:0: module 'vim.uri' not found:
        no field package.preload['vim.uri']
        no file './vim/uri.lua'
        no file '/Users/runner/work/neovim/neovim/.deps/usr/share/luajit-2.1/vim/uri.lua'
        no file '/usr/local/share/lua/5.1/vim/uri.lua'
        no file '/usr/local/share/lua/5.1/vim/uri/init.lua'
        no file '/Users/runner/work/neovim/neovim/.deps/usr/share/lua/5.1/vim/uri.lua'
        no file '/Users/runner/work/neovim/neovim/.deps/usr/share/lua/5.1/vim/uri/init.lua'
        no file './vim/uri.so'
        no file '/usr/local/lib/lua/5.1/vim/uri.so'
        no file '/Users/runner/work/neovim/neovim/.deps/usr/lib/lua/5.1/vim/uri.so'
        no file '/usr/local/lib/lua/5.1/loadall.so'
        no file './vim.so'
        no file '/usr/local/lib/lua/5.1/vim.so'
        no file '/Users/runner/work/neovim/neovim/.deps/usr/lib/lua/5.1/vim.so'
        no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
        [C]: in function 'require'
        vim/_init_packages.lua: in function '__index'
        vim/loader.lua: in function <vim/loader.lua:0>
        [C]: at 0x0104d5dd7c
        [C]: in function 'require'
        vim/_init_packages.lua: in function '__index'
        ...erfer/.local/share/nvim/lazy/lazy.nvim/lua/lazy/init.lua:37: in function 'setup'
        /Users/geoff.langerderfer/.config/nvim/init.lua:67: in main chunk
E484: Can't open file /usr/local/share/nvim/syntax/syntax.vim
Press ENTER or type command to continue
```
## Desktop

- OS: macOS Ventura 13.5
- Terminal: default mac terminal applicationt

## Neovim Version

```
% nvim --version
NVIM v0.10.0-dev-5f4f83b
Build type: RelWithDebInfo
LuaJIT 2.1.1695653777
Run "nvim -V1 -v" for more info
% 
```

**Comment by geofflangenderfer** [comment]
tldr: I copied the neovim runtime files to /usr/local/share/nvim and it fixed it

for whatever reason, my install didn't add the runtime files to the expected location. To fix it, I:
- git cloned neovim
- copied the runtime folder to /usr/local/share/nvim

```
$ git clone https://github.com/neovim/neovim.git
$ git checkout v0.9.1
$ sudo cp -r neovim/runtime /usr/local/share/nvim
```
